recipes/django-rdkit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "django-rdkit" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rdkit/django-rdkit/archive/v{{ version }}.tar.gz
+  sha256: 9b1f13b8777fdb5262af6f8bf1b693d6362604a60514b91643b6ff82e93cc551
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<30]
```

Comment:
You cannot have this line. Use a version constraint directly on the python requirement below.

Reply:
Thanks!
I removed the skip line, but I didn't get where my requirement should go though, could you please explain? I didn't find it in the doc or in the example recipe.